test/e2e/geojson/mapml2geojson.test.js

```diff
@@ -0,0 +1,681 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("GeoJSON API - mapml2geojson", () => {
+    let page;
+    let context;
+    test.beforeAll(async function() {
+      context = await chromium.launchPersistentContext('');
+      page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+      await page.goto("mapml2geojson.html");
+      //await page.waitForTimeout(10000);
+    });
+
+    test.afterAll(async function () {
+      await context.close();
+    });
+
+  test("Point Geometry", async () => {
+    const out0 = await page.$eval(
+      "body > p#json0",
+      (p) => JSON.parse(p.innerHTML)
+    );
+    expect(out0).toEqual(point);
+  });
+
+  test("Line Geometry", async () => {
+   const out1 = await page.$eval(
+     "body > p#json1",
+     (p) => JSON.parse(p.innerHTML)
+   );
+   expect(out1).toEqual(line);
+ });
+
+  test("Polygon Geometry with holes", async () => {
+   const out2 = await page.$eval(
+     "body > p#json2",
+     (p) => JSON.parse(p.innerHTML)
+   );
+   expect(out2).toEqual(polygon);
+ });
+
+  test("MultiPoint Geometry", async () => {
+   const out3 = await page.$eval(
+     "body > p#json3",
+     (p) => JSON.parse(p.innerHTML)
+   );
+   expect(out3).toEqual(multipoint);
+ });
+
+ test("MultiLineString Geometry", async () => {
+   const out4 = await page.$eval(
+     "body > p#json4",
+     (p) => JSON.parse(p.innerHTML)
+   );
+   expect(out4).toEqual(multilinestring);
+ });
+
+ test("MultiPolygon Geometry", async () => {
+   const out5 = await page.$eval(
+     "body > p#json5",
+     (p) => JSON.parse(p.innerHTML)
+   );
+   expect(out5).toEqual(multipolygon);
+ });
+
+ test("Feature Collections", async () => {
+  const out6 = await page.$eval(
+    "body > p#json6",
+    (p) => JSON.parse(p.innerHTML)
+  );
+  expect(out6).toEqual(featurecollection);
+});
+
+test("Geometry Collection", async () => {
+   const out7 = await page.$eval(
+     "body > p#json7",
+     (p) => JSON.parse(p.innerHTML)
+   );
+   expect(out7).toEqual(geometrycollection);
+ });
+
+});
+
+let point = {
+   "title": "Point Geometry",
+   "type": "FeatureCollection",
+   "features": [{
+       "type": "Feature",
+       "geometry": {
+           "type": "Point",
+           "coordinates": [-75.6916809, 45.4186964]
+       },
+       "properties": {
+           "prop0": "This is a Point"
+       }
+   }]
+  };
+
+let line = {

```

Comment:
If you put these variables into a .js file that was loaded by the mapml2geojson.html via `<script src=""></script>`, you could serialize them with stringify into a div that represents the "input" json, then compare the output with that input / expected value.  That way you wouldn't have these variables in here.  Just a suggestion, not sure it will work.